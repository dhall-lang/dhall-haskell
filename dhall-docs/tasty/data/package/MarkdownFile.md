# Title

This is a *Markdown* file.

|A very | nice | table|
|---|---|---|
|1|2|3|
|4|5|6|

