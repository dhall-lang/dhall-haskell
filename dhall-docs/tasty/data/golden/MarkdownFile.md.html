<!DOCTYPE HTML>
<html>
<head>
<title>/MarkdownFile.md</title>
<link rel="stylesheet" type="text/css" href="index.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600;700&amp;family=Lato:ital,wght@0,400;0,700;1,400&amp;display=swap">
<script src="index.js">
</script>
<meta charset="UTF-8">
</head>
<body>
<div class="nav-bar">
<img class="dhall-icon" alt="Dhall logo." src="dhall-icon.svg">
<p class="package-title">test-package</p>
<div class="nav-bar-content-divider">
</div>
<a id="switch-light-dark-mode" class="nav-option">Switch Light/Dark Mode</a>
</div>
<div class="main-container">
<h2 class="doc-title">
<span class="crumb-divider">/</span>
<a href="index.html">test-package</a>
<span class="crumb-divider">/</span>
<span class="title-crumb" href="index.html">MarkdownFile.md</span>
</h2>
<a class="copy-to-clipboard" data-path="/MarkdownFile.md">
<i>
<small>Copy path to clipboard</small>
</i>
</a>
<br>
<details open>
<summary class="part-summary">Rendered content</summary>
<div class="doc-contents">
<h1 id="title">Title</h1>
<p>This is a <em>Markdown</em> file.</p>
<table>
<thead>
<tr>
<th>A very</th>
<th>nice</th>
<th>table</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>2</td>
<td>3</td>
</tr>
<tr>
<td>4</td>
<td>5</td>
<td>6</td>
</tr>
</tbody>
</table>
</div>
</details>
<details>
<summary class="part-summary">Source</summary>
<div class="source-code">
<pre># Title

This is a *Markdown* file.

|A very | nice | table|
|---|---|---|
|1|2|3|
|4|5|6|

</pre>
</div>
</details>
</div>
</body>
</html>